src/components/LaunchFilter/actions/actionTypes.ts

```diff
@@ -0,0 +1,5 @@
+export const enum LaunchActions {
+  LAUNCH_REQUEST = 'LAUNCH_REQUEST',
+  LAUNCH_SUCCESS = 'LAUNCH_SUCCESS',
+  LAUNCH_ERROR = 'LAUNCH_ERROR',
```

Comment:
use convenient name

Reply:
These are templates for action types. They are used as part of the action type name, for example PREVIOUS_LAUNCH_REQUEST, CREWED_LAUNCH_SUCCESS